package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ginger.io/beyonce",
-  "version": "0.0.40",
+  "version": "0.0.41",
```

Comment:
This package is open source, and so I have not bumped it to 1.0x as I don't consider the API to be completely stabilized at this point in time.